### DISCUSSION TITLE
How to adjust lotus-miner actor withdraw packaged speed?

### DISCUSSION THREAD

**Discussion by kevinfitzroy** [filecoin-project/lotus#5499] [General Q&A] [answered]
`lotus-miner actor withdraw`  tx was not packaged and waited for more than a day. What is the solution?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
Solution 1:  set the `DefaultMaxFee` in the daemons `config.toml` to a value that results in a message with a `gas-feecap` that is fitting the chains "current" basefee requirements. see https://docs.filecoin.io/get-started/lotus/configuration-and-advanced-usage/#configuration and the [Fees] section at the bottom for how to set the `DefaultMaxFee`. then go to https://filfox.info/en/stats/gas to get the average gas costs of a withdraw message. then calculate on how to set the fee to get it on chain without being stuck. (https://calc.li.factor8.io has a calculator for message costs if needed)

this will have consequences for other messages!!!

Solution 2: https://docs.filecoin.io/mine/lotus/message-pool/#checking-for-pending-messages --> replacing the message by hand as soon as you trigger it. the message withdraws the amount available at it's creation - there is no gain in having it waiting in line, additional funds will not get withdrawn with the same message.

please notice: `--gas-limit` is an optional parameter in the replace command that is dangerous to touch if you d not know what you are doing -> leave it out of the command
